### DISCUSSION TITLE
Generate Release Notes task in classic release pipeline takes too long

### DISCUSSION THREAD

**Discussion by dev-arma** [rfennell/AzurePipelines#2032] [Q&A] [answered]
I have this working release pipeline in Azure DevOps for a specific site release and I have a stage for release notes using Generate Release Notes task that works just as I want it. However, when I create a new stage with the Generate Release Notes task (or a new pipeline with that includes this task), the **task takes too long to finish**! Whenever this happens, i get `Looking for the [MyBuild] in the most recent successful release [Initial Deployment] ` and it continues to find every build from the start of the project. Is there something I can do to prevent this?

**Comment by rfennell** [comment, marked as answer]
Yes, you can set the parameter `OverrideBuildReleaseId` to the oldest past run you wish the task to consider. it was added for just this issue
